ca/external.go

```diff
@@ -114,7 +114,7 @@ func (eca *ExternalCA) Sign(ctx context.Context, req signer.SignRequest) (cert [
 		cert, err = makeExternalSignRequest(requestCtx, client, url, csrJSON)
 		cancel()
 		if err == nil {
-			return append(cert, eca.rootCA.Intermediates...), err
+			return append(NormalizePEMs(cert), eca.rootCA.Intermediates...), err
```

Comment:
Just wanted to make sure that when the certs are appended together, it's nicely readable since these will be written to disk by the nodes.  Although maybe it makes more sense to just normalize on the node side after it's received.